docs/docs/recipes/organizations/impact-on-end-users.mdx

```diff
@@ -8,4 +8,4 @@ Currently, there's no impact to the end-user sign-in experience since Logto deco
 
 As the user signs in, they can get the access to all organizations with membership. For detailed information, please refer to Organization RBAC.
 
-We may add advanced features for organizations that change the sign-in experience in the future. Please
+We may add advanced features for organizations that change the sign-in experience in the future. Stay tuned! Join our [Discord community](https://discord.gg/UEPaF3j5e6) to share your thoughts and feedback.
```

Comment:
```suggestion
We may add advanced features for organizations that will change the sign-in experience in the future. Stay tuned! Join our [Discord community](https://discord.gg/UEPaF3j5e6) to share your thoughts and feedback.
```

Reply:
"change"  is for "features". "in the feature" modifies "add".